pages/en/using-mina/send-payment.mdx

```diff
@@ -142,3 +142,25 @@ You may not see the `Receipt chain hash` on the first transaction from the accou
 ## Staking and Snarking
 
 Once you feel comfortable with the basics of creating an address, and sending & receiving mina, we can move on to the truly unique parts of the Mina network - <DocLink copy="participating in consensus and helping compress the blockchain" url="/using-mina/staking" />.
+
+## Advanced
+
+### Sending Many Transactions
+
+Sometimes you may wish to send many transactions: For example, to payout rewards to those delegating to you if you're running a staking pool.
+
+All information here is relevant as of the 1.2.0 build (currently in beta):
+
+### Rate limiting
+
+Currently, nodes on the network will rate limit receiving messages from a given node. As of the 1.2.0 build, your node will also follow this rate limit when sending transactions. Specifically, the limit is currently set at 10 transactions every 15 seconds computed over a 5 minute window. If you attempt to send transactions faster than this rate, your node will queue them up and flush them as older transactions expire from the window upon which the rate limit is computed. You do not need to throttle sending these transactions yourself.
```

Comment:
`Note that older releases of the mina daemon do not perform this rate limiting; if you are running an older version, you should manually limit the number of transactions. Due to overheads from rebroadcasting transactions, we do not recommend sending more than 50 transactions every 5 minutes if you need to manually rate limit.`